egs/librispeech/ASR/RESULTS.md

```diff
@@ -1,5 +1,50 @@
 ## Results
 
+### Zipformer CTC
+
+#### [zipformer_ctc](./zipformer_ctc)
+
+See <https://github.com/k2-fsa/icefall/pull/941> for more details.
+
+You can find a pretrained model, training logs, decoding logs, and decoding
+results at:
+<https://huggingface.co/desh2608/icefall-asr-librispeech-zipformer-ctc>
+
+Number of model parameters: 86083707, i.e., 86.08 M
+
+| decoding method         | test-clean | test-other | comment             |
+|-------------------------|------------|------------|---------------------|
+| ctc-decoding            | 2.50       | 5.86       | --epoch 30 --avg 9  |
```

Comment:
Could you also post the result for HLG decoding, i.e., one-best decoding?

Reply:
I think it is strange that 1best (HLG) is better than whole-lattice-rescoring (HLG + 4-gram G).

Reply:
@desh2608 It seems that you don't have a parameter to adjust the scale of the HLG decoding graph. Could you please add this parameter like here: https://github.com/k2-fsa/icefall/blob/05e7435d0d36789ba947b3ca664b730b8d79cb92/egs/librispeech/ASR/conformer_ctc3/decode.py#L250-L254
I tested your model and I got 2.46/5.36 with `hlg_scale=0.5` for 1best decoding.

Reply:
> Yeah, I was thinking the same. I'll verify the numbers again.

Are you able to reproduce it, i.e., WER for test clean = 2.01 ?
@desh2608

Reply:
BTW something else that is different in this recipe compared to other LibriSpeech recipes is that I keep cuts shorter than 25s (instead of 20s), to avoid throwing away more data. With the `quadratic_duration` option in DynamicBucketingSampler, this seems to be working fine (I could train on V100 with batch size 800).